lambdas/handlers/generate_lloyd_george_stitch_handler.py

```diff
@@ -0,0 +1,80 @@
+from boto3.dynamodb.types import TypeDeserializer
+from enums.lambda_error import LambdaError
+from enums.logging_app_interaction import LoggingAppInteraction
+from models.stitch_trace import StitchTrace
+from pydantic import ValidationError
+from services.lloyd_george_generate_stitch_service import LloydGeorgeStitchService
+from utils.audit_logging_setup import LoggingService
+from utils.decorators.ensure_env_var import ensure_environment_variables
+from utils.decorators.handle_lambda_exceptions import handle_lambda_exceptions
+from utils.decorators.override_error_check import override_error_check
+from utils.decorators.set_audit_arg import set_request_context_for_logging
+from utils.lambda_exceptions import LGStitchServiceException
+from utils.lambda_response import ApiGatewayResponse
+from utils.request_context import request_context
+
+logger = LoggingService(__name__)
+
+
+@ensure_environment_variables(
+    names=[
+        "LLOYD_GEORGE_DYNAMODB_NAME",
+        "LLOYD_GEORGE_BUCKET_NAME",
+        "STITCH_METADATA_DYNAMODB_NAME",
+    ]
+)
+@set_request_context_for_logging
+@override_error_check
+@handle_lambda_exceptions
+def lambda_handler(event, context):
+    request_context.app_interaction = LoggingAppInteraction.DOWNLOAD_RECORD.value
+
+    logger.info("Triggered by Dynamo INSERT event")
+
+    dynamo_records = event.get("Records")
+
+    if not dynamo_records:
+        logger.error("No records in event")
+        raise LGStitchServiceException(400, LambdaError.StitchClient)
```

Comment:
Maybe the name records is misleading, this is when the lambda is triggered but the 'event' json object is missing 'Records' field in it. So it would a 400.